.env.example

```diff
@@ -1,2 +1,2 @@
-POSTGRES_DB=example
+POSTGRES_DB=game
 POSTGRES_PASSWORD=qMrYxNBR
```

Comment:
So I'm guessing this is an intentionally insecure password for local dev and not the real Heroku DB password, but I would try not to be in the habit of ever checking in passwords. Just because git is designed to make it really hard to delete things, you really never want to check in a password because then it can't be used for anything secure again.

I'm not sure about your Heroku setup, but if you need help I'm happy to look at the setup and help you get the passwords set up properly.